app/assets/stylesheets/redde/redactor/index.scss

```diff
@@ -147,8 +150,8 @@ body .redactor_toolbar li a.redactor_btn_justifyright span		  { background-posit
 body .redactor_toolbar li a.redactor_btn_justify span		 	  { background-position: -450px; }
 body .redactor_toolbar li a.redactor_btn_horizontalrule span 	  { background-position: -475px; }
 body .redactor_toolbar li a.redactor_btn_fullscreen span		  { background-position: -500px; }
-body .redactor_toolbar li a.redactor_btn_format span		  { background-position: -575px; }			
-body .redactor_toolbar li a.redactor_btn_normalscreen span		  { background-position: -525px !important; }	
+body .redactor_toolbar li a.redactor_btn_format span		  { background-position: -575px; }
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 5<br>Selector `redactor_toolbar` should be written in lowercase with hyphens<br>Selector `redactor_btn_format` should be written in lowercase with hyphens<br>Opening curly brace `{` should be preceded by one space